lib/OpenQA/WebAPI/Plugin/ObsRsync/templates/ObsRsync_folder.html.ep

```diff
@@ -0,0 +1,69 @@
+% layout 'bootstrap';
+% content_for 'head' => begin
+  %= asset 'codemirror.js'
+  %= asset 'codemirror.css'
+% end
+
+% title 'OBS synchronization';
+<H2><%= 'OBS synchronization for: ' . $obs_project  %></H2>
+<div class="container" id="content">
+    <p>OBS synchronization is performed in 3 steps.</p>
+</div>
+<hr>
+<div class="container" id="content">
+<H6>Step 1. Retrieve lists of required files from predefined OBS locations.</H6>
+% if (@$lst_files) {
+        <label>No files have been read so far</label>
+% } else {
+        <label>Lists retrieved during last run: </label>
+% }
+% for my $filename (@$lst_files) {
+        <label><%= link_to "$filename"  => url_for('plugin_obs_rsync_download_file', folder => $obs_project, subfolder => ".run_last", filename => $filename) %></label>
+        <label>|</label>
+% }
+<br>
+% if ($read_files_sh) {
+        <label>Script used to retrieve files: </label>
+	<label><%= link_to "$read_files_sh"  => url_for('plugin_obs_rsync_download_file', folder => $obs_project, subfolder => ".run_last", filename => $read_files_sh) %></label>
+% }
+</div>
+<hr>
+<div class="container" id="content">
+<H6>Step 2. Generate and execute commands to rsync and rename deliverables from remote OBS location (based on lists from Step 1.)</H6>
+% if (!scalar(@$rsync_commands)) {
```

Comment:
`scalar` is redundant here, the context is enforced by the `!` operator already.

Reply:
But not everybody who possibly reads the code knows this. So to improve general readability it actually makes sense to clearly state that the array should be converted to a scalar.